tests/test_rero_invenio_files.py

```
@@ -170,3 +170,42 @@ def test_files_api_flow(app, client, headers, file_location, pdf_file):
     res = client.get(f"/api/records/{id_}/files", headers=headers)
     assert res.status_code == 200
     assert len(res.json["entries"]) == 0
+
+
+def test_file_download_content_disposition(app, client, headers, file_location, pdf_file):
```

Ruff (0.14.14)

Unused function argument: `app`

(ARG001)